beetsplug/gmusic.py

```diff
@@ -57,11 +63,22 @@ def commands(self):
 
     def upload(self, lib, opts, args):
         items = lib.items(ui.decargs(args))
-        files = [x.path.decode('utf-8') for x in items]
+        files = self.getpaths(items)
         ui.print_(u'Uploading your files...')
         self.m.upload(filepaths=files)
         ui.print_(u'Your files were successfully added to library')
 
+    def autoupload(self, session, task):
+        items = task.imported_items()
+        files = self.getpaths(items)
+        self._log.info(u'Uploading files to Google Play Music...', files)
+        self.m.upload(filepaths=files)
+        self._log.info(u'Your files were successfully added to your '
+                  + 'Google Play Music library')
```

Comment:
Travis has a silly whitespace complaint about this line:
https://travis-ci.org/beetbox/beets/jobs/412848100